tdrs-backend/tdpservice/security/test/test_owasp_zap_scan.py

```diff
@@ -0,0 +1,54 @@
+"""Unit tests for OWASP ZAP scan operations."""
+from django.contrib.admin.models import LogEntry
+import pytest
+
+from tdpservice.security.models import OwaspZapScan
+from tdpservice.users.models import User
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
```

Comment:
Leverages [attribute fixtures](https://pytest-factoryboy.readthedocs.io/en/latest/#attributes-are-fixtures) created by pytest-factoryboy to provide multiple field overrides for each test run.